v1/plugin/plugin.go

```diff
@@ -280,67 +342,70 @@ func buildGRPCServer(typeOfPlugin pluginType, name string, version int, opts ...
 // generates a response for the initial stdin / stdout handshake, and starts
 // the plugin's gRPC server.
 func StartCollector(plugin Collector, name string, version int, opts ...MetaOpt) int {
-	server, m, err := buildGRPCServer(collectorType, name, version, opts...)
+	appArgs.plugin = plugin
+	appArgs.name = name
+	appArgs.version = version
```

Comment:
When `-h` is run, there is a section `usage` that has a generic message in it still: `NAME: snap-plugin-collector rand - a new cli application`
We should change `a new cli application` to be `a Snap collector plugin`. This is where getPluginType() was being used previously.